CONTRIBUTING.md

```diff
@@ -51,12 +50,23 @@ and now we can run the local Jupyter Lab and run the scripts etc. using Poetry:
 
 - Create an issue to discuss the feature's scope and its fit for this package
 - run pytest to ensure your local version of code passes all unit tests
+```
+pytest tests/unit
+```
+
 - try to edit the relevant code and implement your new feature in a backwards compatible manner
 - create new tests as you go, and run the test suite as you go
 - update the documentation as you go
 
 ### Please format and lint as you go with the following scripts
 
+In order to run the following scripts, run the following,
+```
+pip install mypy
+mypy --install-types
+pip install ruff
+```
+Then run these scripts to ensure correctx format, style and convention:
```

Comment:
These are already installed by `poetry install` as instructed earlier in this guide. Also, the commands below should be replaced with `make lint format`, which is what I've done in this PR (in review): https://github.com/nsidc/earthaccess/pull/497

Reply:
In my experience, I was unable to get these things working post `poetry install` from a fresh build. If we want to consolidate on `make <test|lint|format>` I'm cool with that.

Reply:
What problem did you encounter, because this works for me, as given at the top of the file (although with `conda`, not `mamba`)?

```plain
conda env update -f ci/environment-dev.yml
conda activate earthaccess-dev
poetry install
```